src/utils/CL.ipynb

```diff
@@ -16,10 +16,59 @@
   },
```

Comment:
remove this also from this project. Its a Giza experiment, not relevant to our official work at strkfarm as of now, unless there is something useful